kotlin-vercel-ncc/build.gradle.kts

```diff
@@ -0,0 +1,10 @@
+plugins {
+    `kotlin-library-conventions`
+}
+
+dependencies {
+    jsMainImplementation(libs.coroutines.core)
```

Comment:
Redundant dependency

Reply:
Yeah, now with `@JsAsync` it is, thanks :-)